app/src/main/java/org/oppia/app/help/faq/FAQActivity.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.app.help.faq
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The FAQ page activity for placement of different FAQs. */
+class FAQActivity : InjectableAppCompatActivity() {
+
+  @Inject
+  lateinit var FAQActivityPresenter: FAQActivityPresenter
```

Comment:
Update:
`private lateinit var faqActivityPresenter: FAQActivityPresenter`

Reply:
I am keeping this as this:
`@Inject lateinit var faqActivityPresenter : FAQActivityPresenter` 

Else I am getting the error `lateinit property faqActivityPresenter has not been initialized`